### PR TITLE
Feature/payment

### PR DESCRIPTION
**Pull request by liayoo**
- Allow multiple addresses to be service admin for payments
- Remove id fields from payment records
- Update tests

### CODE REVIEW

File: genesis-configs/base/genesis_rules.json [hunk 1]
```diff
@@ -81,23 +81,24 @@
   "payments": {
     "$service": {
       "config": {
-        ".write": "evalOwner('/payments/' + $service + '/config', 'write_owner', auth) && data === null && util.isDict(newData)"
+        "admin": {
+          "$admin_addr": {
+            ".write": "evalOwner('/payments/' + $service + '/config', 'write_owner', auth) && util.isBool(newData)"
```
Comment (platfowner) [inline genesis-configs/base/genesis_rules.json:86]: Suggestion: What about `newData === true` unless `false` value has special meaning?

Comment (platfowner) [reply genesis-configs/base/genesis_rules.json:86]: For deletion, `newData === true || newData === null`?

Comment (liayoo) [reply genesis-configs/base/genesis_rules.json]: Right, actually updated the rule but didn't commit 🤦‍♀️ Thanks!!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**